common/models/workspace.js

```diff
@@ -350,13 +350,19 @@ module.exports = function(Workspace) {
         });
 
         // Wait until the child process starts listening
-
         var waitOpts = {
           host: host,
           port: port || 3000, // 3000 is the default port provided by loopback
           timeoutInMs: 30000  // 30 seconds
         };
 
+        // Windows will fail to connect if the host is 0.0.0.0, so redirect to
+        // localhost to prevent the startup detection from timing out.
+        if (waitOpts.host === '0.0.0.0') {
+          waitOpts.host = 'localhost';
```

Comment:
FWIW, here is the LoopBack code dealing with this problem:

https://github.com/strongloop/loopback/blob/bfd5059d119d7a7131a04c9873cdc61f04aa1165/lib/application.js#L519-L547